SkietbaanBE/SkietbaanBE/Controllers/TestController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Test")]
+    public class TestController : Controller
+    {
+        // GET: api/Test
+        [HttpGet]
+        public string Get()
+        {
+            return "Connected to SkietBaan Web Api";
```

Comment:
I really don't think that's necessary, since this is simply a test string which will never be used anywhere else.

It would make sense in a front-end project though, especially if the site needed to be served in different languages.

Reply:
> I really don't think that's necessary, since this is simply a test string which will never be used anywhere else.
> 
> It would make sense in a front-end project though, especially if the site needed to be served in different languages.

i guess i will have to remove the entire controller. it was just to allow  the front end to test api connection without having to accesss the information from the database